pkg/proxy/endpoint.go

```diff
@@ -154,3 +170,29 @@ func convertToMap(events []string) map[string]bool {
 
 	return eventsMap
 }
+
+func convertToMapAndSanitize(headers []string) map[string]string {
+	reg, err := regexp.Compile("[\x00-\x1f]+")
+
+	headerMap := make(map[string]string)
+
+	if err != nil {
+		// what to do if regex fails to compile?
```

Comment:
`panic(err)` https://gobyexample.com/panic

Reply:
So turns out I can use "regexp.MustCompile" which automatically panics instead of returning an error!